docs/user-interface/splashscreen.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Splash screens"
+description: ".NET Multi-platform App UI (.NET MAUI) splash screens can be displayed when an app is launched, while the app's initialization process completes."
+ms.date: 08/19/2021
+---
+
+# Splash screens
+
+<!-- Todo: Move this content into the images doc, once it migrates over? -->
+
+On iOS and Android, .NET Multi-platform App UI (.NET MAUI) apps can display a splash screen while their initialization process completes. The splash screen is displayed immediately when an app is launched, providing immediate feedback to users while app resources are initialized. Once the app is ready for interaction, the splash screen is dismissed.
+
+Splash screens are a composite of an image and a background color, which can both be customized. The standard platform image formats are supported, including Scalable Vector Graphics (SVG) files.
+
+> [!TIP]
+> The SVG format is the recommended image format for .NET MAUI splash screens.
+
+.NET MAUI splash screens are specified by dragging an image into the **Resources** folder of your project, and setting the build action of the image to **MauiSplashScreen** in the **Properties** window. This creates a corresponding entry in the .csproj file for your project:
```

Comment:
Is there an **Images** subfolder? the example below seems to indicate there is.

Reply:
The template adds the splash screen to the Resources folder, rather than Resources\Images. I've logged a bug against the template, in the hope it gets updated. I'll update the text to say to put the splash screen in Resources\Images, and will add a todo comment to check it again before launch.